### PR TITLE
Remove old spectacle cruft.

### PR DESCRIPTION
**Pull request by spiiroin**
We don't use spectacle anymore.

Remove the yaml-file and associated cruft.

### CODE REVIEW

File: rpm/libiphb.spec [hunk 1]
```diff
@@ -84,8 +61,6 @@ rm -rf %{buildroot}
 %defattr(-,root,root,-)
 %{_libdir}/libiphb.so.*
 %doc COPYING AUTHORS INSTALL README NEWS ChangeLog
```
Comment (saukko) [inline rpm/libiphb.spec]: We should probably move most of these to -devel only, no need to have these on normal system.

Comment (spiiroin) [inline rpm/libiphb.spec]: Something like this perhaps?
%doc COPYING %{?qa_stage_devel:AUTHORS INSTALL README NEWS ChangeLog}

Comment (thp) [inline rpm/libiphb.spec]: IMHO let's just keep them in instead of doing qa-stage-dependend conditionals here.

**Comment by thp** [comment]
LGTM. Thanks for cleaning up the Spectacle cruft!

**Pull request merged**